apps/frontend/src/libs/components/balance-wheel-chart/libs/helpers/draw-sublabels/draw-sublabels.helper.ts

```diff
@@ -0,0 +1,62 @@
+import { HALF_PI } from "~/libs/constants/constants.js";
+import { type ChartArea, type RadialLinearScale } from "~/libs/types/types.js";
+
+import {
+	ChartFont,
+	ChartGraphicsColor,
+	LabelDistacneOffset,
+	WheelCenterDistance,
+} from "../../enums/enums.js";
+import { getBaseline } from "../get-baseline/get-baseline.helper.js";
+import { getGraphicsOffset } from "../get-graphics-offset/get-graphics-offset.js";
+import { getSublabelOffsetX } from "../get-sublabel-offset-x/get-sublabel-offset-x.helper.js";
+import { getSublabelOffsetY } from "../get-sublabel-offset-y/get-sublabel-offset-y.helper.js";
+
+const drawSublabels = ({
+	chartArea,
+	context,
+	label,
+	middleAngle,
+	scale,
+}: {
+	chartArea: ChartArea;
+	context: CanvasRenderingContext2D;
+	label: string;
+	middleAngle: number;
+	scale: RadialLinearScale;
+}): void => {
```

Comment:
Let's move these to a separate type ChartSublabelsData or your own name. Intension is just the same as above

Reply:
I noticed that I have same data for both sublabels and labels, so I created common type `ChartLabelsData` for both sublabels and value-labels